### DISCUSSION TITLE
兼容啊B的H5API

### DISCUSSION THREAD

**Discussion by SSSSSSSan** [xfgryujk/blivechat#186] [想法]
啊B的H5API会加上鉴权参数和页面参数，其中plug_env==0是给主播直接一键添加到直播姬或者在幻星页面单击复制地址时候的实际使用的页面，plug_env==1则代表着希望显示配置界面，经过测试在配置页面内是可以完成样式文件上传操作的（至少在iframe嵌套下可以）
配合现在的“导入服务器预设CSS”，使用主播身份码识别身份，在配置页面上传样式文件后，在plug_env==0页面直接加载对应的样式文件也不是不行
而且流量都经过啊b转发，隐藏了真实的服务器域名和ip 岂不美哉
况且现在只用了不到50行php就能直接用上了
![{CE34761F-A964-40f2-88EB-F8E89CBFCA7D}](https://github.com/xfgryujk/blivechat/assets/40064285/47f30f0b-4a13-45bf-b34a-94473dd89eb0)

**Comment by xfgryujk** [comment]
有想法是好事，但也只能想想
